django/contrib/admin/widgets.py

```diff
@@ -131,7 +131,7 @@ def url_params_from_lookup_dict(lookups):
             else:
                 v = str(v)
             items.append((k, v))
-        params.update(dict(items))
+        params.update(items)
```

Comment:
Could `items` not be removed completely in favour of `params[k] = v`?

Reply:
Makes sense.